common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
```

Comment:
@andy-armstrong, ideas about what to do here? Is this the right way to go about getting UXPL stuff in v1 CSS?